pubspec.yaml

```diff
@@ -130,6 +130,7 @@ dependencies:
   # fk_user_agent
   fk_user_agent: 2.1.0
 
+  firebase_analytics: 9.1.2
```

Comment:
why we should add analytic also?

Reply:
I think there should be so we can see the log of events and actions from firebase. This firebase library is also recommended.